packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -76,7 +76,9 @@ export function extractLogSafeErrorProperties(
 }
 
 /**
- * type guard for ILoggingError interface
+ * Type-guard for {@link @fluidframework/core-interfaces#ILoggingError}.
+ *
+ * @public
```

Comment:
Eh... I'm on the fence here. I thought this would be good (so consumers can leverage `getTelemetryProperties`) but some internal partners haven't shown interest in that, so dunno. Probably leave it public for now.

Reply:
Are any existing partners using this that you know of? We've opted to aggressively mark anything that isn't used by partners already as `@internal` until we have a reason to expose it.

Reply:
Just checked and no in bohemia nor loopsdk. Probably ok to make internal.